docs/dev/developer-guides/transactions/blocks.md

```diff
@@ -59,10 +64,36 @@ Block hashes in zkSync are deterministic and are derived from the following form
 The reason for having a deterministic block hash is that these hashes are not provable (remember that L2 blocks are not submitted to L1).
 Projects are advised not to use the L2 block hash as a source of randomness.
 
-### Block Properties
+## Block Properties
+
+## block.number and block.timestamp
+
+Depending on where it's used, if `block.number` and `block.timestamp` is retrieved via the `zksync-web3` SDK, miniblocks values are returned. It is a special construct on zkSync Era to make sure that the transactions are processed promptly. This approach is similar in other L2s, where each transaction belongs to a separate block, (which we call miniblocks), while these miniblocks are gathered together into batches that get sent to L1.
+
+In Solidity, `block.number` and `block.timestamp` returns the number and timestamp of the L1 batch respectively.
+
+### zksync-web3 SDK
+
+The following are the block properties returned when you use the `getBlock` method via the **zksync-web3** API.
+
+| Parameter     | Description                                                                                                             |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------|
+| hash          | The hash of the block.                                                                                                  |
+| parentHash    | It refers to the hash of the parent block in L2.                                                                        |
+| number        | the number of the current block                                                                                         |
+| timestamp     | The current block's creation time in seconds.                                                                           |
+| nonce         | It's the most recent transaction based on the account's counter, which maintains track of how many transactions it does.|
+| difficulty    | The current block difficulty returns 2500000000000000 (zkSync does not have proof of work consensus).                   |
+| gasLimit      | The block gas limit, always returns `2^32-1`.                                                                           |
+| gasUsed       | The actual amount of gas used in this block.                                                                            |
+| transactions  | A list of all transactions included in the block.                                                                       |
```

Comment:
please, change the description to smth like An array of transaction objects - please see [eth_getTransactionByHash](link to SDK method) for exact shape

Reply:
@AnastasiiaVashchuk which of the property are you referring to in your statement?